test-helpers/src/startFixture/webpack.ts

```diff
@@ -112,7 +112,7 @@ export const startWebpackFixture = (
             url: `http://localhost:${port}`,
             close: () =>
               new Promise<void>((resolveClose) =>
-                server.close(() => {
+                server.stopCallback(() => {
```

Comment:
`close` was replaced with the async `stop` and callback variant `stopCallback`. https://github.com/webpack/webpack-dev-server/blob/c3b532c6360317319793dcda22c76fbfc05fbdcf/migration-v5.md